backend/glp/tests/test_review_fixes.py

```
@@ -0,0 +1,246 @@
+from .helpers import (
+    give_builder_points,
+    make_program,
+    make_season,
+    make_user,
+    run_request,
+)
+
+CRON_TOKEN = 'test-cron-token'
+
+
+def _approve_and_execute(request):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_approve_and_execute`

(ANN202)